src/components/Input/style.js

```diff
@@ -12,11 +12,48 @@ import {
 import { fontFamily, fontSize, fontWeight, lineHeight } from '../style/fonts';
 import Text from '../Text';
 
+const inputPadding = css`
```

Comment:
I had never seen the `css` reference before. So glad we're using styled components now :)